hs_core/models.py

```diff
@@ -3525,7 +3525,7 @@ def parse_for_bulk_update(cls, metadata, parsed_metadata):
             parsed_metadata.append({"language": {"code": metadata.pop('language')}})
 
         if 'rights' in keys_to_update:
-            parsed_metadata.append({"rights": {"statement": metadata.pop('rights')}})
```

Comment:
@sblack-usu Removing this ```statement``` nesting might potentially break landing page rights display. For example, https://github.com/hydroshare/hydroshare/blob/344f4590c8615fd7ffd67e4c04930e23b6504147/theme/templates/resource-landing-page/citation.html#L34

Reply:
this is only for the rest api and was incorrectly implemented.  The internal urls pass in statement and url.